<launch>
    <arg name="is_show" default="true" />
    <arg name="use_stereo" default="true"/>
    <param name="use_sim_time" value="true" type="bool" />
    <node pkg="swarm_detection" name="swarm_detection" type="swarm_detection_node" output="screen">
        <remap from="~left_camera" to="/mynteye/left/image_raw"/>
        <remap from="~right_camera" to="/mynteye/right/image_raw"/>
        <param name="is_show" type="bool" value="$(arg is_show)" />
        <param name="use_stereo" type="bool" value="$(arg use_stereo)"/>
        <param name="duration" type="double" value="0.001" />
        <param name="left_cam_def" type="string" value="/home/xuhao/mf2_home/SwarmConfig/mini_mynteye_stereo/left.yaml" />
        <param name="right_cam_def" type="string" value="/home/xuhao/mf2_home/SwarmConfig/mini_mynteye_stereo/right.yaml" />
        <param name="vo_def" type="string" value="/home/xuhao/mf2_home/SwarmConfig/mini_mynteye_stereo/mini_mynteye_stereo_imu.yaml" />
    </node>
    <node pkg="rosbag" type="play" name="player" output="screen" args="--clock /home/xuhao/bags/swarm-detection/far_stereo_2019-03-18-11-31-14.bag"/>
</launch>
